tests/main/mount-ns/task.yaml

```diff
@@ -0,0 +1,112 @@
+summary: the shape of the mount namespace on classic systems for non-classic snaps
```

Comment:
Nitpick: we are not super consistent with this, but I think we generally start with capital letters in task summaries.

Reply:
Just propose a suggestion please. I'll gladly take it.

Reply:
```suggestion
summary: The shape of the mount namespace on classic systems for non-classic snaps
```